pkg/expr/func_time.go

```diff
@@ -195,3 +196,27 @@ func strfTime(base interface{}, f string) (string, error) {
 	o, _ := strftime.Format(f, *t, strftime.WithMilliseconds('b'))
 	return o, nil
 }
+
+// sub returns difference between two date into milliseconds
+func sub(from interface{}, to interface{}) (out int64, err error) {
+	t1, _, err := prepMod(from, 0)
+	if err != nil {
+		return
+	}
+
+	t2, _, err := prepMod(to, 0)
+	if err != nil {
+		return
+	}
+
+	var duration time.Duration
+	if t1.Before(*t2) {
+		duration = t2.Sub(*t1)
+	}
+
+	if t1.After(*t2) {
+		duration = t1.Sub(*t2)
+	}
+
+	return duration.Microseconds(), nil
```

Comment:
The comment states miliseconds but here you do microseconds; do milliseconds as further accuracy is not needed.